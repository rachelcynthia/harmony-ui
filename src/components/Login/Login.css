.login_container {
    display: flex;
    position: absolute;
    top: 0;
    bottom: 0;
    right: 0;
    left: 0;
    margin-top: 60px;
    background: linear-gradient(90deg, rgba(255, 255, 255, 1) 0%, rgba(203, 108, 230, 1) 44%, rgba(207, 118, 232, 1) 50%, rgba(255, 255, 255, 1) 100%);
}

.login_component {
    margin: auto;
    height: 600px;
    width: 500px;
    background-color: black;
    color: aliceblue;
    border: 5px solid #cb6ce6;
    border-radius: 20px;
}

.login_header {
    text-align: center;
    font-family: Apple Chancery, cursive;
    font-size: 40px;
    letter-spacing: 1.5px;
    padding: 10px 0px;
    border-bottom: 5px solid #cb6ce6;
}

.login_body {
    margin-top: 30px;
    row-gap: 30px;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

.input_text {
    height: 40px;
    width: 75%;
    text-align: center;
    font-family: Blippo, fantasy;
    letter-spacing: 2px;
    font-weight: 600;
    color: black;
    font-size: 15px;
    /* -webkit-transition: 0.5s;
    transition: 0.5s;
    outline: none; */
}

.input_text:focus {
    border-color: rgba(203, 108, 230, 0.78);
    border-width: 2.5px;
    border-style: solid;
    outline: none !important;
}

.input_button {
    cursor: pointer;
    width: 60%;
    height: 50px;
    font-family: Blippo, fantasy;
    font-size: 20px;
    vertical-align: middle;
    letter-spacing: 2px;
    font-weight: 600;
    color: black;
}

.input_button:focus {
    border-color: rgba(203, 108, 230, 0.78);
    border-width: 2.5px;
    border-style: solid;
    outline: none !important;
}

.reg_component {
    font-family: DejaVu Sans Mono, monospace;
    font-size: 18px;
    margin-top: 30px;
    text-align: center;
}

.register_link {
    color: #cb6ce6;
}

.register_button {
    font-family: DejaVu Sans Mono, monospace;
    font-size: 18px;
    color: white;
}