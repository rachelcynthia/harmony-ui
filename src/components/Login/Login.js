import React from 'react';
import './Login.css'

const Login = () => {
    const handleSubmit = () => {
        return true;
    }
    return (
        <div className='login_container'>
            <div className='login_component'>
                <div className='login_header'>
                    Please login to continue
                </div>
                <div className='login_body'>
                    <input type="text" placeholder='Enter email' className="input_text" />
                    <input type="password" placeholder='Enter password' className="input_text" />
                    <input type="button" onClick={handleSubmit()} className="input_button" value="LOGIN" />
                </div>
                <div className='register_component'>
                    New to Harmony? <a className='register_button' href='#'>Register</a>
                </div>
            </div>

        </div>
    );
}
export default Login