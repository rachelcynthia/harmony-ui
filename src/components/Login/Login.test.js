import React from 'react';
import TestRenderer from 'react-test-renderer';

import Login from './Login';

it('renders correctly', () => {
    const login = TestRenderer.create(<Login />).toJSON();
    expect(login).toMatchSnapshot();
});