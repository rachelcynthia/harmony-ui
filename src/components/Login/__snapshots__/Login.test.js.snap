// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login_container"
>
  <div
    className="login_component"
  >
    <div
      className="login_header"
    >
      Please login to continue
    </div>
    <div
      className="login_body"
    >
      <input
        className="input_text"
        placeholder="Enter email"
        type="text"
      />
      <input
        className="input_text"
        placeholder="Enter password"
        type="password"
      />
      <input
        className="input_button"
        onClick={true}
        type="button"
        value="LOGIN"
      />
    </div>
    <div
      className="register_component"
    >
      New to Harmony? 
      <a
        className="register_button"
        href="#"
      >
        Register
      </a>
    </div>
  </div>
</div>
`;
